examples/basic/outputs.tf

```diff
@@ -0,0 +1,23 @@
+##############################################################################
+# Outputs
+##############################################################################
+
+output "region" {
+  description = "The region all resources were provisioned in"
+  value       = var.region
+}
+
+output "prefix" {
+  description = "The prefix used to name all provisioned resources"
+  value       = var.prefix
+}
+
+output "resource_group_name" {
+  description = "The name of the resource group used"
+  value       = var.resource_group
```

Comment:
this output should come from the module output

Reply:
addressed